examples/declarative_withAuthToken

```diff
@@ -0,0 +1,37 @@
+pipeline {
+    agent any
+
+    stages {
+        stage ('Stage 1') {
+            steps {
+                echo "First stage OK"
+            }
+        }
+        stage('Wait on Webhook') {
+            options {
+                timeout(time: 3, unit: "MINUTES")
+            }
+            steps {
+                script {
+                    withCredentials([string(credentialsId: 'webhook_secret', variable: 'SECRET')]) { 
+
+                        hook = registerWebhook(token: 'test-webhook', authToken: SECRET)
+                        callbackURL = hook.getURL()
+                        sh "echo ${callbackURL}"
+
+
+                        echo "Waiting for POST to ${callbackURL}"
```

Comment:
This line seems to be a duplicate with the previous one `sh echo ${callbackURL}`

Reply:
👍 
I should improve this. The first echo is where the external system is started and provided with the URL to call when its processing is completed. It is indeed not clear from the example.